pom.xml

```diff
@@ -94,7 +94,7 @@
     <joda.version>2.9.9</joda.version>
     <hadoop.version>2.7.3</hadoop.version>
     <hive.groupid>org.apache.hive</hive.groupid>
-    <hive.version>2.3.1</hive.version>
+    <hive.version>2.3.6</hive.version>
```

Comment:
Why are we updating Hive?

Reply:
When running hive sync through Spark 2.x, it will use [hive-spark](https://github.com/JoshRosen/hive/tree/release-1.2.1-spark2) as hive dependency which is ```1.2.1-spark2``` version. So I need to make sure all the Hive APIs used in ```HoodieHiveClient``` are compatible with both hive-spark ```1.2.1-spark2``` and Hive ```2.3.x```.

Actually for this alter_partition API: ```client.alter_partition(String, String, Partition)```. I didn't find a compatible API between 2.3.1 and 1.2.1. But I can find a compatible API with Hive 2.3.6 and hive-spark 1.2.1.

So I am thinking of bumping Hive version to 2.3.6. Is this acceptable to the community?